hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSStripedInputStream.java

```diff
@@ -554,4 +555,17 @@ public synchronized void releaseBuffer(ByteBuffer buffer) {
     throw new UnsupportedOperationException(
         "Not support enhanced byte buffer access.");
   }
+
+  @Override
+  public synchronized void unbuffer() {
+    closeCurrentBlockReaders();
```

Comment:
Overall looks good. Just one nit.
Could you call super.unbuffer() instead?

+1 after that.